src/main/java/mate/jdbc/service/CarServiceImpl.java

```diff
@@ -0,0 +1,61 @@
+package mate.jdbc.service;
+
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        return carDao.get(id);
+    }
+
+    @Override
+    public List<Car> getAll() {
+
+        try {
+            return carDao.getAll();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        return carDao.delete(id);
+    }
+
+    @Override
+    public Car update(Car car) {
+        return carDao.update(car);
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        return carDao.getAllByDriver(driverId);
+    }
+
+    public void addDriverToCar(Driver driver, Car car) {
+        carDao.addDriverToCar(driver, car);
+    }
+
+    @Override
+    public void removeDriverFromCar(Driver driver, Car car) {
+        carDao.removeDriverFromCar(driver, car);
+    }
```

Comment:
those methods should be only in service layer
![image](https://user-images.githubusercontent.com/81781479/199534112-099b781f-8039-4cdc-a812-04055b4515e3.png)